libs/dh/admin/feature-user-management/src/lib/drawer/dh-edit-user-modal/dh-edit-user-roles-tab.component.ts

```diff
@@ -0,0 +1,29 @@
+﻿/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { WattCardModule } from '@energinet-datahub/watt/card';
+import { TranslocoModule } from '@ngneat/transloco';
+
+@Component({
+  selector: 'dh-edit-user-roles-tab',
+  standalone: true,
+  imports: [CommonModule, WattCardModule, TranslocoModule],
+  templateUrl: './dh-edit-user-roles-tab.component.html',
+  styles: [``],
+})
+export class DhEditUserRolesTabComponent {}
```

Comment:
nitpick: this component doesn't seem to be used.

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1374/commits/3f88cdb5ead6f7ee96b0ce84b0a4eff8bb5facc7